dev_guide/expose_service/expose_internal_ip_service.adoc

```diff
@@ -343,16 +343,16 @@ eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
         ...
 ----
 
-. Add a route between the IP address of the host where the master resides and the gateway IP address of the master host:
+. Add a route between the IP address of the host where the master resides and the gateway IP address of the master host. If using a netmask for a networking route, use the `netmask` option, as well as the netmask to use:
```

Comment:
I'd drop this change... the other places don't bother mentioning it.

Reply:
@knobunc That's what the BZ is about, so I've kept the change, but extended it to the other examples. Let me know if you disagree.